drivers/google/google_test.go

```diff
@@ -0,0 +1,153 @@
+package google
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/codegangsta/cli"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDriverName(t *testing.T) {
+	driverName := newDriver().DriverName()
+
+	assert.Equal(t, "google", driverName)
+}
+
+func TestFailWithoutProject(t *testing.T) {
+	_, err := create([]string{"test-name"})
+
+	assert.EqualError(t, err, "Please specify the Google Cloud Project name using the option --google-project.")
+}
+
+func TestParseFlags(t *testing.T) {
+	driver, err := create([]string{
```

Comment:
I beg to differ. The issue we are to fix comes from code that fails to parse the command line. If we unit test `SetConfigFromFlags` with a `DriverOptionsMock`, we don't spot the bug. There needs to be something that verifies that a given command line transforms into the appropriate action or error. An e2e test would cost too much, would rely on external services and couldn't cover all edge cases. By calling the `create` method with all the combinations of arguments, we can almost test 100% of the driver and this is cheap and doesn't rely on anything external.